src/nifgen/examples/nifgen_script.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+
+import argparse
+import nifgen
+import numpy as np
+from scipy import signal
+import sys
+import time
+
+
+def calculate_sinewave():
+    time = np.arange(0, 10, 0.01)                 # np.arange(Start, Stop, Step)
+    amplitude = np.sin(time)
+    sinewave = amplitude.tolist()                 # List of Float
+    return sinewave
+
+
+def calculate_rampup():
+    ramp = np.arange(0, 0.5, 0.01)                # numpy.arange([start,]stop, [step,]dtype=None(
+    ramp_up = ramp.tolist()                       # List of Float
+    return ramp_up
+
+
+def calculate_rampdown():
+    ramp = np.arange(0, 0.5, 0.01)                # numpy.arange([start,]stop, [step,]dtype=None)
+    ramp_down = ramp.tolist()                     # List of Float
+    ramp_down.reverse()                           # Reverse list to get a ramp down
+    return ramp_down
+
+
+def calculate_square():
+    time = np.arange(0, 10, 0.01)                 # np.arange(Start, Stop, Step)
+    square_build = signal.square(time, 0.5)       # signal.square(time,duty_cycle)
+    square = square_build.tolist()                # List of Float
+    return square
+
+
+def calculate_triangle():
+    time = np.linspace(0, 1, 200)                 # np.arange(Start, Stop, Step)
```

Comment:
3d parameter is not Step, it's num.
I think for all these, if you use named parameters then this sort of copy/paste error goes away and the need for the comment disappears.

Reply:
Switched to named parameters.

Fixed comments, some add information about defaults, so I kept them.